formatter-gitlab/README.md

```diff
@@ -0,0 +1,121 @@
+# markdownlint-cli2-formatter-gitlab
+
+> An output formatter for [`markdownlint-cli2`][markdownlint-cli2] that writes
+> results to a [GitLab code quality report artifact][gitlab] [JSON][json] file
+> (a subset of the [CodeClimate spec][codeclimate])
+
+[![npm version][npm-image]][npm-url]
+[![License][license-image]][license-url]
+
+## Install
+
+```shell
+npm install markdownlint-cli2-formatter-gitlab --save-dev
+```
+
+## Use
+
+For the default output file name of `"markdownlint-cli2-results.json"`, use
+the following `.markdownlint-cli2.jsonc`:
+
+```json
+{
+  "outputFormatters": [
+    [ "markdownlint-cli2-formatter-gitlab" ]
+  ]
+}
+```
+
+To customize the output file name or number of spaces to indent, use the
```

Comment:
Is indent a meaningful parameter here? I am thinking it should not be because the purpose of this file is to be ingested by some other tool. It was only included with the JSON formatter because some people might have used that to review visually.

Reply:
Same comment for output file name. If folks wouldn't need to customize it, let's leave it out.

Reply:
Good point. I removed both of them.